api/Insight_API_documentation.md

```diff
@@ -0,0 +1,1200 @@
+
+| Parameter           | Type                   |  Description                   | 
+| -------------------- | ---------------------- | ---------------------- | 
+| hash              | `string`      |   Address    |  
+
+**Request Example:**
+
+```GET /txs?address=DseRvgwcGeCJJC4AUWimHSj8YzRMJ4ryvBM```
+
+**Request Response:**
+
+```
+{
+    "pagesTotal": 21782,
+    "txs": [
+        {
+            "txid": "9ba978f7bf8cde1163c84ca4b249d0a97a090ba549d186407cea51f21aee50f2",
+            "version": 1,
+            "locktime": 0,
+            "vin": [
+                {
+                    "txid": "97fcf72fc252c2d6f535a4d5be8055acc67d7e72bbc8dc64fd5445b8bce9ded7",
+                    "vout": 2,
+                    "sequence": 4294967295,
+                    "n": 0,
+                    "scriptSig": {
+                        "hex": "47304402204aa300044486b74f0c6bd6a44ae7e2001ae48c644604aa75c41506d6614ba51602205b227085fb9935ed64ff7772ecb5fd4ec791fee8279cd6b6aef8ff7001f73f2d0121032fa170be0502f9049d63a1c7c6be2f16ced286be6dc06fedf83520380ae2e674",
+                        "asm": "304402204aa300044486b74f0c6bd6a44ae7e2001ae48c644604aa75c41506d6614ba51602205b227085fb9935ed64ff7772ecb5fd4ec791fee8279cd6b6aef8ff7001f73f2d01 032fa170be0502f9049d63a1c7c6be2f16ced286be6dc06fedf83520380ae2e674"
+                    },
+                    "addr": "DsaCBHbg9PX7qbn32oK2aUxYhENdTgpdcJK",
+                    "valueSat": 9032520744,
+                    "value": 90.32520744
+                },
+                {
+                    "txid": "0dc66ade626754745f23b915e63102d3edb59180a074f383c5fef741b42af3f8",
+                    "vout": 2,
+                    "sequence": 4294967295,
+                    "n": 1,
+                    "scriptSig": {
+                        "hex": "483045022100b683ac2b7668840dc048944407aa941e8f2e026f1d3a708bf7dfe0eea799700502206488d9da5301893672b97412f6dd27657c108682c6b7feefca15f45f0e739b45012102f8857d1459fb56fe3ba0f006943f2ec785f5e4bbd1d2480d0a1af2e8875214da",
+                        "asm": "3045022100b683ac2b7668840dc048944407aa941e8f2e026f1d3a708bf7dfe0eea799700502206488d9da5301893672b97412f6dd27657c108682c6b7feefca15f45f0e739b4501 02f8857d1459fb56fe3ba0f006943f2ec785f5e4bbd1d2480d0a1af2e8875214da"
+                    },
+                    "addr": "Dsbr3zGyBcDYWWibnENAhW3N8UwJ5J4tnKg",
+                    "valueSat": 8965893473,
+                    "value": 89.65893473
+                },
+.
+.
+.
+.
+
+            ],
+            "blockhash": "0000000000000008537e82b35ac3e7b4549289cfb440e1218260ef3fceb7770e",
+            "blockheight": 229806,
+            "confirmations": 49336,
+            "time": 1523675017,
+            "blocktime": 1523675017,
+            "valueOut": 4248.47935122,
+            "size": 10043,
+            "valueIn": 4248.48951622,
+            "fees": 0.010165
+        },
+
+    ]
+}
+```
+
+<br/>
+
+### /tx/send/ (POST)
+
+**URL:**  ```POST /tx/send {"rawtx":[RAWTXDATA]"}``` **REVIEW**
+
```

Comment:
Not really part of the URL.  the parameters are posted as part of the body as a JSON.   I think the curl example is good.